templates/login.html.twig

```diff
@@ -6,7 +6,7 @@
             <div class="col-md-6" align="center">
                 <h2>Please sign in</h2>
                 <form method="post" action="">
```

Comment:
w action na wszelki wypadek nalezy dac url do strony /login

Reply:
Dodanie /login powoduje internal server error. Dodanie ścieżki ../templates/login.html.twig powoduje page not found, po przesłaniu forma z prawidowymi danymi.

Reply:
wykonaj najpierw wszystkie zmiany ktore podalem